modules/nw-ingress-controller-configuration-proxy-protocol.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-proxy-protocol_{context}"]
+= Configuring PROXY protocol for an Ingress Controller
+As a cluster administrator, you can use PROXY protocol to enable communication between the Ingress Controller and an external load balancer. An Ingress Controller is typically published with a load balancer. The connections that the Ingress Controller receives are from the load balancer, so the source address has the load balancer address, instead of the original connection address. The PROXY protocol enables a load balancer to preserve the original source addresses and include them when forwarding connections to an Ingress Controller, which can then use the original source addresses for logging.
+
+The ability to configure PROXY protocol is useful when the Ingress Controller specifies the `HostNetwork` or `NodePortService` endpoint publishing strategies, which integrate with external load balancers.
```

Comment:
To be clear,  `HostNetwork` or `NodePortService` _can_ be used with external load-balancers, but it is possible to use `HostNetwork` or `NodePortService` without a load balancer.

Reply:
@Miciah ok, I removed the bit about external load balancers and adjusted the sentence a little.